tools/autograd/gen_python_functions.py

```diff
@@ -15,7 +15,7 @@
 # These functions require manual Python bindings or are not exposed to Python
 SKIP_PYTHON_BINDINGS = [
     'alias', 'contiguous', 'clamp.*', 'is_cuda', 'is_sparse', 'size', 'stride',
-    'slice_dim', '.*_backward'
+    '.*_backward', '.*_backward_out', '.*_forward', '.*_forward_out',
```

Comment:
why did we not skip .*_forward before?  Mistake?

Reply:
I think it was excluded due to some other black magic.